Sources/Extensions/Foundation/FileManagerExtensions.swift

```diff
@@ -56,5 +56,19 @@ public extension FileManager {
         return nil
     }
 
+    /// SwifterSwift: Save Encodable object to a given URL.
+    ///
+    /// - Parameters:
+    ///   - object: Object of type T that conforms to Encodable protocol.
+    ///   - url: URL to save object to.
+    ///   - encoder: JSONEncoder.
+    /// - Throws: Throws any errors thrown by JSONEncoder or FileManager.
+    public func encode<T: Encodable>(_ object: T, to url: URL, using encoder: JSONEncoder = JSONEncoder(), attributes: [FileAttributeKey: Any]? = nil) throws -> Bool {
+        let data = try encoder.encode(object)
+        if createFile(atPath: url.path, contents: data, attributes: attributes) {
```

Comment:
Why not just call:
```suggestion
        return createFile(atPath: url.path, contents: data, attributes: attributes)
```
?

Reply:
@guykogus, To perform a second check (Check that file contains the expected JSON value), I am assuming I need to decode the file? (I will add a decode() extension method in another PR so I don't add more than 1 extension in this one)